src/main/resources/io/jenkins/plugins/designlibrary/Banner/index.jelly

```diff
@@ -1,29 +1,48 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples">
-  <s:sample>
-    <div class="jdl-leading-paragraph">
-      Banners are a handy way to display small snippets of text and actions.
-    </div>
-    <div class="jdl-component-group">
-      <div class="jdl-component-sample jdl-component-sample--full-width jdl-component-sample--gridless">
-        <div class="jenkins-alert jenkins-alert-info">
-          Hello world
-        </div>
-        <div class="jenkins-alert jenkins-alert-warning">
-          Hello world
-        </div>
-        <div class="jenkins-alert jenkins-alert-danger">
-          Hello world
-        </div>
-        <div class="jenkins-alert jenkins-alert-success jenkins-!-margin-bottom-0">
-          Hello world
-        </div>
-      </div>
-      <s:code language="xml" file="index.jelly" />
-    </div>
-    <div class="jenkins-alert jenkins-alert-warning">
```

Comment:
do we still need this warning? its not that old at this point?

Reply:
Restored it.